packages/ui/src/components/EasySetup/ManualExtrinsic.tsx

```diff
@@ -13,19 +14,22 @@ import { ISubmittableResult } from '@polkadot/types/types'
 import React, { ChangeEvent, useCallback, useEffect, useMemo, useState } from 'react'
 import { useApi } from '../../contexts/ApiContext'
 import paramConversion from '../../utils/paramConversion'
-import { getTypeDef } from '@polkadot/types/create'
+// import { getTypeDef } from '@polkadot/types/create'
```

Comment:
to remove

Reply:
I was on the fence with this one, because it's super hard to find :smile: my IDE didn't add it automatically, and so because it goes with the other comments (the console.logs that are pretty useful to quickly un-comment rather than find and re-create) I left it. 

Let's remove it, and hope it's not too hard to find again.
```suggestion
```